.github/workflows/cancel-duplicates.yml

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Cancelling Duplicates
+on:
+  workflow_run:
+    workflows: 
+      - 'CI - Build - MacOS'
```

Comment:
is it necessary to write here the list of jobs ?
we can forget about it in the future

Reply:
The documentation is not very precise and does not describe whether this field is optional. I have not seen any examples that do not define it, so I think it is required. On the other hand, this event doesn't support the exclude list, so it seems to be the only way to exclude "PulsarBot".
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_run

Reply:
Unfortunately I think this is the only way :( . What you can do is to make some pre-commit check to verify if the list is complete/